specs/_features/eip7594/p2p-interface.md

```diff
@@ -93,6 +97,9 @@ def verify_data_column_sidecar_inclusion_proof(sidecar: DataColumnSidecar) -> bo
     """
     Verify if the given KZG commitments included in the given beacon block.
     """
+    # A sidecar for zero blobs is invalid
+    assert len(sidecar.kzg_commitments) > 0
```

Comment:
do we think this is a duplicate check here?

it may not be valid outside the scope of the function, but you can make a perfectly valid inclusion proof for a `sidecar` with 0 blobs

Reply:
I think this logic also applies for verifying kzg proofs; ie its valid to pass no commitments and proofs and the cryptography function would handle it appropriately.

I guess one alternative would be for the caller of these methods to do the empty sidecar checks themselves before calling these methods